engines/python/setup/djl_python/rolling_batch/lmi_dist_rolling_batch.py

```diff
@@ -283,5 +283,5 @@ def pin_lora(self, lora_name):
         # 2) An adapter is not evicted, call add_lora() is not necessary.
         # But since whether an adapter is evicted is not exposed outside of engine,
         # and add_lora() in this case will take negligible time, we will still call add_lora().
-        self.engine.add_lora(lora_request)
-        return self.engine.pin_lora(lora_request.lora_int_id)
+        return self.engine.add_lora(lora_request) and self.engine.pin_lora(
+            lora_request.lora_int_id)
```

Comment:
do we need to check the result of add_lora before pinning?

Also, I would prefer if we kept these calls separate. It's more readable.

Sames questions for vlm_rolling_batch.py

Reply:
1) This is to make sure it's successfully loaded before pinning.
2) Made these calls separate.